### ISSUE TITLE
Bright and normal colours are identical

### ISSUE THREAD

**Issue by ojdo** [pineapplegiant/spaceduck-terminal#8]
I very much enjoy the overall look of this colour scheme, but had to disable it due to normal and bright colours being identical. This correcting commit 00eaad35db62b554094774d48e12f9fb3ba21775 makes me believe that this is not accidental but intentional. Is there a chance to have all 16 colours distinct?

Breaking for me is the mail view in MUA [alot](https://github.com/pazz/alot), for which all mail header lines become invisible with SpaceDuck active.

**Comment by ojdo** [comment]
Here is a pair of screenshots with the difference in appearance between Campbell and SpaceDuck:

### Campbell

![grafik](https://user-images.githubusercontent.com/3525666/131250539-bb3bc346-3885-4224-bb92-871a4e756766.png)

### SpaceDuck

![grafik](https://user-images.githubusercontent.com/3525666/131250550-e4fe94ba-b90f-4aa8-ac27-dd299a244d99.png)

And just for reference, why it occurs in alot: its default theme uses [dark and light gray](https://github.com/pazz/alot/blob/a814a7744e0e7d98656980fcdae3ed712a299d8e/alot/defaults/default.theme#L47-L51) for foreground/background of `header_value` (i.e. the field contents).

**Comment by kmuncie** [comment]
I would agree, there should be a distinction between normal and bright colors. I ended up adjusting the Windows Terminal color scheme in order to resolve the issues I was having. Here is the config I ended on and some examples from before and after through neofetch.

```json
        {
            "name": "SpaceDuck",
            "cursorColor": "#f2ce00",
            "selectionBackground": "#f2ce00",
            "foreground": "#ecf0c1",
            "background": "#0f111b",
            "black": "#000000",
            "red": "#d36240",
            "green": "#5ccc96",
            "yellow": "#b3a1e6",
            "blue": "#6d3c8a",
            "purple": "#c0aa2e",
            "cyan": "#00a3cc",
            "white": "#c9cca9",
            "brightBlack": "#686f9a",
            "brightRed": "#e33400",
            "brightGreen": "#51eea2",
            "brightYellow": "#9d79ff",
            "brightBlue": "#9630d1",
            "brightPurple": "#f2ce00",
            "brightCyan": "#19c5f0",
            "brightWhite": "#fafce0"
        }
```

Repo color scheme:
![before](https://user-images.githubusercontent.com/582755/154856342-0bf55564-31e5-48fa-b1e1-959e299b7c79.png)

My custom color scheme changes:
![after2](https://user-images.githubusercontent.com/582755/154856357-02138c5b-ffbc-4e20-a922-9500261aef89.png)

**Comment by pineapplegiant** [comment]
This looks pretty good. I'll have to test and add this in. If it works, it'd be good to update all the other configs as well. I've heard some complaints from others regarding this issue. Thanks for this @kmuncie ! 🚀 🦆
